File: core-java-practice/gcr-codebase/programming-elements/PowerCalculation.java
package Dayone;
import java.util.*;
public class PowerCalculation {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    // take the base input
    double base = sc.nextDouble();
    // take the exponent input
    double exponent = sc.nextDouble();
    // calculate the result 
    double result = Math.pow(base, exponent);
    //print the result 
    System.out.println("Result = " + result);
  }
}

File: core-java-practice/gcr-codebase/programming-elements/PowerCalculation.cs
using System;
namespace ProgrammingElements{
    class PowerCalculation{
        static void Main(string[] args){
            // take the base input
            double baseValue = double.Parse(Console.ReadLine());
             // take the exponent input
            double exponent = double.Parse(Console.ReadLine());
            // calculate the result
            double result = Math.Pow(baseValue, exponent);
            // print the result
            Console.WriteLine("Result = " + result);
        }
    }
}
